backends-common/elasticsearch-v7/src/test/java/org/apache/james/backends/es/v7/ElasticSearchIndexerTest.java

```diff
@@ -80,6 +81,7 @@ void tearDown() throws IOException {
     }
 
     @Test
+    @Disabled
```

Comment:
we usually put the message in @Disabled that refers to the ticket number linked to it. Here, you can use the JIRA 3492 so that we can search remaining tests to fix easily

Reply:
Added ticket number to Disabled tags